src/main/java/com/rackspacecloud/metrics/ingestionservice/utils/InfluxDBFactory.java

```diff
@@ -0,0 +1,9 @@
+package com.rackspacecloud.metrics.ingestionservice.utils;
+
+import org.influxdb.InfluxDB;
+
+public class InfluxDBFactory {
+    public InfluxDB getInfluxDB(String url) {
+        return org.influxdb.InfluxDBFactory.connect(url);
```

Comment:
I would have shifted all of https://github.com/racker/ceres-ingestion-service/pull/5/files/ccc7604d4a83f6829d3545c7586f766dd89e8b93..7d786becb258ed73dac913bc02f8fabb2d174b0f#diff-aff45cee99c0b2218cf7dc6301959b4fL269 into this method, which is actually what I did in the event engine ingest

https://github.com/racker/salus-event-engine-ingest/blob/master/src/main/java/com/rackspace/salus/event/ingest/services/KapacitorConnectionPool.java#L32

however, mine was simpler since I didn't yet need to tweak batch options.

Reply:
Agree and done.